resources/definitions/creality_ender3.def.json

```diff
@@ -59,6 +59,9 @@
         "jerk_travel": {
             "value": "jerk_print"
         },
+        "machine_max_jerk_z": {
+            "default_value": 0.3,
```

Comment:
This doesn't parse as JSON because of the extra comma at the end.

Did you test this in Cura?

Reply:
No I haven't tested it, I tried to edit my local file but couldn't find a way to do so. I'm very new at this.

Reply:
If you're on Windows you could edit the file at `C:\Program Files\Ultimaker Cura 4.0\resources\definitions\creality_ender3.def.json`, but you need administrator rights. If you're on Linux the AppImage is an ISO image that you could edit. If you're on MacOS the distribution is also an ISO image but since it is signed you'd need to disable the signature checking for it to run after editing it.

Since the impact of this setting change is minor and it has a good basis in theory I think I'll just accept the change like this without you testing it.